csharp/ql/src/change-notes/2022-02-23-extractor-option-compression.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* The C# extractor now accepts an extractor option `trap.compression`, which is used to decide the compression format for TRAP files. The legal options are `brotli` (default), `gzip` or `none`.
+* The C# extractor no longer accepts `--no-brotli` or `--brotli` flags to switch between `gzip` and `brotli` as compression method for TRAP files.
```

Comment:
I don't think we need to document this, since these are arguments directly to `Semmle.Extraction.CSharp.Driver.exe`, which is not something that anyone is expected to invoke manually.